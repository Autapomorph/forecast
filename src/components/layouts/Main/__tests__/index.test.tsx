import React from 'react';
import { shallow } from 'enzyme';

import MainLayout from '..';

describe('MainLayout tests', () => {
  it('renders properly', () => {
    const wrapper = shallow(<MainLayout />);

    expect(wrapper).toMatchSnapshot();
  });
});
