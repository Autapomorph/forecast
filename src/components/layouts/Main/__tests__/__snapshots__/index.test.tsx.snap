// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout tests renders properly 1`] = `
<styles__LayoutWrapper>
  <ToastContainer
    autoClose={5000}
    bodyClassName={null}
    className={null}
    closeButton={
      <CloseButton
        ariaLabel="close"
      />
    }
    closeOnClick={true}
    draggable={true}
    draggablePercent={80}
    hideProgressBar={false}
    newestOnTop={false}
    pauseOnFocusLoss={true}
    pauseOnHover={true}
    position="bottom-center"
    progressClassName={null}
    progressStyle={null}
    role="alert"
    rtl={false}
    style={null}
    toastClassName={null}
    transition={[Function]}
  />
  <styles__HeaderWrapper>
    <Header />
  </styles__HeaderWrapper>
  <styles__ContentWrapper>
    <Connect(City) />
    <Connect(Cities) />
    <Connect(Featured) />
    <Modal
      close={[Function]}
      isOpen={false}
    />
  </styles__ContentWrapper>
</styles__LayoutWrapper>
`;
